### PR TITLE
updated for TLD 2.03

### PR DESCRIPTION
**Pull request by ajkroeg**
name is t-bone on TLD modding discord

### CODE REVIEW

File: FasterSearching.csproj [hunk 1]
```diff
@@ -1,69 +1,36 @@
...
+	<!--This is the of packages that the mod references.-->
+	<ItemGroup>
+		<!--This package contains almost everything a person could possibly need to reference while modding.-->
+		<PackageReference Include="ds5678.Modding.TLD.Il2CppAssemblies.Windows" Version="2.6.0" />
+		<!--If the mod references any other mods (such as ModSettings), their NuGet package will also need to be listed here.-->
+	</ItemGroup>
+	<ItemGroup>
+	  <Reference Include="ModSettings">
+	    <HintPath>..\..\..\..\..\..\SteamLibrary\steamapps\common\TheLongDark\Mods\ModSettings.dll</HintPath>
+	  </Reference>
```
Comment (ds5678) [inline FasterSearching.csproj]: ModSettings has a nuget package.

**Review by ds5678 — CHANGES_REQUESTED** [review]
Hello, I am a friend of @zeobviouslyfakeacc, and I have push permissions for most of his repositories. I briefly looked over your pull request, and only one thing stuck out at me. If you make the change I mentioned, I'll merge this.

However, I don't want to put much effort into maintaining this, so I won't post a release unless you also implement continuous integration. I have an example at https://github.com/ds5678/SeasonedInterloping/blob/main/.github/workflows/build.yml

**Comment by ds5678** [comment]
Add a workflow dispatch to the continuous integration. You can see an example on the AssetRipper/Tpk repository.

SeasonedInterloping also has a dependabot file. Copy that over to this.

Your csproj mod template is slightly out of date. Please update it.

**Comment by ds5678** [comment]
I edited SeasonedInterloping to have workflow dispatch.

**Review by ds5678 — APPROVED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    - name: Build
+      run: dotnet build /p:Configuration=Release
+
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        name: FasterSearching
+        path: ./FasterSearching/bin/Release/net6.0/FasterSearching.dll
+        if-no-files-found: error
```
Comment (ds5678) [inline .github/workflows/main.yml]: I checked on your repository, and the build action is failing in this step.

**Review by ds5678 — CHANGES_REQUESTED** [review]

**Comment by zeobviouslyfakeacc** [comment]
Try `path: ./bin/Release/net6.0/FasterSearching.dll` 😃

**Review by ds5678 — APPROVED** [review]

**Pull request merged**